packages/desktop/src/renderer/components/Channel/Channel.tsx

```diff
@@ -101,8 +101,13 @@ const Channel = () => {
       return updatedFiles
     })
   }
-  const handleClipboardFiles = (imageBuffer, ext) => {
-    const id = `${Date.now()}_${Math.random().toString(36).substring(0, 20)}`
+  const handleClipboardFiles = (imageBuffer, ext, name) => {
+    let id: string
+    if (name === 'image') {
```

Comment:
Why do we need this?

Reply:
Why is that a fixed string instead of const/enum?

Reply:
when we copy image from disc we have image name in clipboard, but when we copy image from other places like browser or when we do a screenshot then every image in clipboard will have same name "image.png"

Reply:
ok, got it. It's fine then I suppose

Reply:
maybe some comment with what you just said could be helpful